src/components/layout/SectionLayout/sectionLayout.scss

```diff
@@ -14,17 +14,20 @@
   &__top {
     display: flex;
     flex: 1;
+    max-height: 354px; // Max height that the usual image on webpage can grow
```

Comment:
I think it is not a good idea to set this max-height static like this, since we are building the section layout that can be used on many places. A better idea could be to receive a custom maxHeight as optional prop in the component.

Reply:
Yeah, I will change it to `fit-content`.